File: clients/sbotop/streams/src/main/java/io/symplify/sqs/Body.java
package io.symplify.sqs;

import io.quarkus.runtime.annotations.RegisterForReflection;

@RegisterForReflection
public class Body<T> {

  public String type;
  public String contactId;
  public String mappingSelector;
  public T properties;

  public Body(){}
}

File: clients/solar/solar-stream/src/main/scala/com/jada/models/Body.scala
package com.jada.models

import io.circe.Encoder
import io.circe.generic.semiauto._

case class Body[T](
    `type`: String,
    contactId: String,
    mappingSelector: String,
    properties: T
)

object Body {
  implicit def bodyEncoder[T: Encoder]: Encoder[Body[T]] = deriveEncoder
}
